src/cmd/gz.cc

```diff
@@ -188,19 +261,19 @@ extern "C" int runServer(const char *_sdfString,
   // Initialize console log
   if ((_recordPath != nullptr && std::strlen(_recordPath) > 0) ||
     _record > 0 || _recordResources > 0 || _recordPeriod >= 0 ||
-    (_recordTopics != nullptr && std::strlen(_recordTopics) > 0))
+    (_recordTopics.size() > 0))
   {
     if (_playback != nullptr && std::strlen(_playback) > 0)
     {
       gzerr << "Both record and playback are specified. Only specify one.\n";
-      return -1;
+      return;
```

Comment:
consider a non-void return type in order to communicate these errors to the calling function

Reply:
cef74ed29